src/simulation/simulation.jl

```diff
@@ -276,6 +280,88 @@ function _initialize_simulation_state!(sim::Simulation)
     return
 end
 
+function _get_model_store_requirements!(
+    rules::CacheFlushRules,
+    model::OperationModel,
+    num_rows::Int,
+)
+    model_name = get_name(model)
+    horizon = get_horizon(model)
+    reqs = SimulationModelStoreRequirements()
+    container = get_optimization_container(model)
+
+    for (key, array) in get_duals(container)
+        reqs.duals[key] = _calc_dimensions(array, key, num_rows, horizon)
+        add_rule!(rules, model_name, key, false, CachePriority.LOW)
+    end
+
+    for (key, param_container) in get_parameters(container)
+        array = get_parameter_array(param_container)
+        reqs.parameters[key] = _calc_dimensions(array, key, num_rows, horizon)
+        add_rule!(rules, model_name, key, false, CachePriority.LOW)
+    end
+
+    for (key, array) in get_variables(container)
+        reqs.variables[key] = _calc_dimensions(array, key, num_rows, horizon)
+        add_rule!(rules, model_name, key, false, CachePriority.HIGH)
```

Comment:
Does `keep_in_cache` need to be true here?

Reply:
fixed via e10ee0e73c0e9c1ab6b1ee4a384c1b02c5899829